### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by efekarakus**
Add three new issue templates: 
* 🐛Bug report
* 💡Feature request
* ❤️ Say thank you

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/---bug-report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+### Summary
+<!-- Please provide a brief outline of the issue -->
+
+### Description
+<!-- Provide detailed information about this issue, including: -->
+* Which specific command was run? (Example: `ecs-cli up --capability-iam --size 2 --launch-type EC2`)
+* Which version of the CLI you are using? (Run: `ecs-cli --version`)
+* Which version of Go are you using? (Run: `go version`)
+* What platform are you using to run ECS CLI commands? (E.g. Linux, MacOSX, Windows)
```
Comment (kohidave) [inline .github/ISSUE_TEMPLATE/---bug-report.md]: nit, Mac is now `macOS`

File: .github/ISSUE_TEMPLATE/---bug-report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Please note that GitHub issues are public, remove sensitive data from your config files before posting.
+If you are not comfortable posting your files here, please let us know and we can provide an alternate method.
+-->
+* docker-compose.yml
+
+* ecs-params.yml
+
+* ~/.ecs/config
+
+<!-- Not required for feature requests -->
```
Comment (kohidave) [inline .github/ISSUE_TEMPLATE/---bug-report.md]: Do we need this comment now that we have a separate template?

File: .github/ISSUE_TEMPLATE/---bug-report.md (hunk 1 above)
Comment (efekarakus) [reply .github/ISSUE_TEMPLATE/---bug-report.md]: Done!

File: .github/ISSUE_TEMPLATE/---bug-report.md (hunk 2 above)
Comment (efekarakus) [reply .github/ISSUE_TEMPLATE/---bug-report.md]: Removed 👍

File: .github/ISSUE_TEMPLATE/---say-thank-you.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+name: "❤️ Say thank you"
+about: "Tell us how you use ecs-cli \U0001F60D"
+title: ''
+labels: Love
```
Comment (iamhopaul123) [inline .github/ISSUE_TEMPLATE/---say-thank-you.md]: Is it really an issue🤷‍♂️?

File: .github/ISSUE_TEMPLATE/---feature-request.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+name: "\U0001F4A1 Feature request"
+about: "Suggest an idea for this project \U0001F914"
+title: ''
+labels: Feature request
+assignees: ''
+
+---
+
+**Context**
```
Comment (iamhopaul123) [inline .github/ISSUE_TEMPLATE/---feature-request.md]: Nitpick: will it be better if we have the same font size for the subtitles in all these three files?

File: .github/ISSUE_TEMPLATE/---say-thank-you.md (hunk 3 above)
Comment (efekarakus) [reply .github/ISSUE_TEMPLATE/---say-thank-you.md]: 😄 it's inspired by https://github.com/nock/nock/issues/new/choose to get feedback on how folks use the tool

File: .github/ISSUE_TEMPLATE/---feature-request.md (hunk 4 above)
Comment (efekarakus) [reply .github/ISSUE_TEMPLATE/---feature-request.md]: Sure thing, changed all the headers to h4 to be consistent

File: .github/ISSUE_TEMPLATE/---feature-request.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+about: "Suggest an idea for this project \U0001F914"
+title: ''
+labels: Feature request
+assignees: ''
+
+---
+
+#### Context
+
+What are you trying to do and how would you want to do it differently? Is it something you currently you cannot do? Is this related to an issue/problem?
```
Comment (hencrice) [inline .github/ISSUE_TEMPLATE/---feature-request.md]: "Is it something you currently you cannot do?" -> "Is it something you currently cannot do?"

"Is this related to an issue/problem?" what data are we trying to gather for this question? Do we want the requester to provide a Github issue number?

Comment (efekarakus) [reply .github/ISSUE_TEMPLATE/---feature-request.md]: 1. Fixed the typo :)
2. Yeah, taken from: https://github.com/nock/nock/issues/new?template=02_feature_request.md, my guess is these questions are to get as much context as possible about the feature request
